README.md

```diff
@@ -131,6 +131,22 @@ $myCopy = $deepCopy->copy($myObject);
 // $myCopy->category has not been touched
 ```
 
+### `DataFilter`
+
+If you want modify data of property like value of an array you can use the `DataFilter`.
+
+```php
+$deepCopy();
+$callback = function($data){ $data['options']['option1'] = false; return $data;};
```

Comment:
It's a detail but maybe you could put the callback on several lines for readability?